tasks/clear-cache-reference-data.js

```diff
@@ -0,0 +1,57 @@
+const redisStore = require('../config/redis-store.js')
+
+const scanAndDelete = async (cursor, pattern) => {
+  const client = await redisStore().client
```

Comment:
`client` doesn't return a promise, so I don't think you need the `await` when defining this variable.